src/Psl/Shell/ErrorOutputBehavior.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Shell;
+
+enum ErrorOutputBehavior {
+    /**
+     * Discard the standard error output.
+     */
+    case DISCARD;
+
+    /**
+     * Append the standard error output content to the standard output content.
+     */
+    case APPEND;
+
+    /**
+     * Prepend the standard error output content to the standard output content.
+     */
+    case PREPEND;
+
+    /**
+     * Replace the standard output content with the standard error output content.
+     */
+    case REPLACE;
+}
```

Comment:
feature behaviors to include:

- merge: merge both stdout and stderr, the order is not guaranteed, as both streams will be read concurrently, and which ever returns first will be first, this doesn't mean that one will come after the over, as stdout can write a chunk, then stderr, then stdout, so it will be mixed ( exactly what you see in your terminal when stderr is not redirected )
- redirect: redirect the stderr output of the process, to the stderr output of the current process.

Reply:
#335 introduces a streaming function which helps implement the merge, and redirect behaviors.